testnet/src/node.rs

```diff
@@ -116,9 +98,6 @@ fn spawn_peer(mut this: PeerNetwork, p2p_sock: &SocketAddr, rpc_sock: &SocketAdd
                 }
             };
 
-            // TODO: take the NetworkResult this method spits out and feed it into a Relayer
-            // instance running in another thread (which will, in turn, take care of storing the
-            // blocks, processing them, and relaying new data to neighbors).
             this.run(&mut burndb, &mut chainstate, &mut mem_pool, None, poll_timeout)
```

Comment:
Just flagging this here -- this method returns a `NetworkResult` that has to be fed into a `net::relay::Relayer` instance, which will (a) process and store newly-discovered blocks and (b) queue up `BlocksAvailable`, `MicroblocksAvailable`, `Microblocks`, and `Transaction` messages for broadcast.  This has implications in the run-loop -- namely, the run-loop _cannot_ be responsible for calling `process_blocks()`, and will instead need to depend on the relayer to give it the latest chain tip metadata and transaction receipts.